spec/intl.html

```diff
@@ -436,7 +438,7 @@ <h1>FormatDateTimePattern ( _dateTimeFormat_, <ins>_pattern_</ins>, _patternPart
             1. Append a new Record { [[Type]]: _p_, [[Value]]: _fv_ } as the last element of the list _result_.
           1. Else if _p_ is equal to *"timeZoneName"*, then
             1. Let _f_ be _dateTimeFormat_.[[TimeZoneName]].
-            1. Let _v_ be _dateTimeFormat_.[[TimeZone]].
+            1. Let _v_ be <del>_dateTimeFormat_.[[TimeZone]]</del><ins>_timeZone_</ins>.
```

Comment:
Isn't this incorrect for cases where _timeZone_ is undefined?

Reply:
Oh yes! I'll fix this.

Reply:
If we go with the suggestion from my previous comment, then _timeZone_ can't be undefined here anymore.